openedx_lti_tool_plugin/resource_link_launch/ags/tests/test_tasks.py

```diff
@@ -222,18 +197,13 @@ def test_send_problem_score_update(
             datetime_mock.now.return_value,
         )
 
-    @patch('openedx_lti_tool_plugin.tasks.log')
-    def test_send_problem_score_update_without_graded_resource(
+    @patch(f'{MODULE_PATH}.log')
+    def test_without_graded_resource(
         self,
         log_mock: MagicMock,
         lti_graded_resource_mock: MagicMock,
     ):
-        """Test send_problem_score_update task without graded resources.
-
-        Args:
-            log_mock: mocked log.
-            lti_graded_resource_mock: mocked LtiGradedResource model.
-        """
+        """Test without graded resources."""
```

Comment:
@kuipumu Why did you remove the description of the arguments?

Reply:
@anfbermudezme I removed them because it's very clear that they are all mock objects, also I think this affects the readability of the tests